index.html

```diff
@@ -51,6 +51,109 @@ <h1 class="introduction">I'm Ivan<br>Glad to see you</h1>
+                    <img src="./images/javascript.png" alt="javascript tag" class="work-tags">
+                </div>
+                <a href="#" class="see-project"><img src="./images/Enabled.svg" alt="Button that links to the project"></a>
+            </div>
+        <div class="work">
+            <img src="./images/work3.svg" alt="screenshot of first project" class="work1_pic">
+            <h3 class="work-name">Tonic</h3>
+            <div class="work-data">
+                <strong class="company"> Canopy </strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="role">Back-End Dev</strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="year">2015</strong>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <img src="./images/html.png" alt="html tag" class="work-tags">
+                <img src="./images/css.png" alt="css tag" class="work-tags">
+                <img src="./images/javascript.png" alt="javascript tag" class="work-tags">
+            </div>
+            <a href="#" class="see-project"><img src="./images/Enabled.svg" alt="Button that links to the project"></a>
+        </div>
+        <div class="work">
+            <img src="./images/work4.svg" alt="screenshot of first project" class="work1_pic">
+            <h3 class="work-name">Multi-Post Stories</h3>
+            <div class="work-data">
+                <strong class="company"> Canopy </strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="role">Back-End Dev</strong>
+                <img src="./images/Counter.png" class="counter" alt="counter">
+                <strong class="year">2015</strong>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="tags">
+                <img src="./images/html.png" alt="html tag" class="work-tags">
+                <img src="./images/css.png" alt="css tag" class="work-tags">
+                <img src="./images/javascript.png" alt="javascript tag" class="work-tags">
+            </div>
+            <a href="#" class="see-project"><img src="./images/Enabled.svg" alt="Button that links to the project"></a>
+        </div>
+    </section>
+    <section class="about">
+        <h2 class="about-me">About<br>Myself</h2>
+        <p class="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="connect-about">Let's connect</a>
```

Comment:
- [ ] Please make sure that this and all titles or subtitles are embraced by a `<hx>` tag, so this will help your code to be more clear and semantic.

Reply:
Do you mean "Let's connect"? I did not think this was a title or subtitle, that is why I did not embrace it with the tag.

Reply:
Yes. Here I'm requiring to add "LET'S CONNECT" into an `<hx>` tag.
You can consider the following:
 - Any piece of text that it's not a paragraph and if it is kinda important that the user takes a look at it (more than a normal paragraph) you can embrace it with an `<hx>` tag.

Reply:
Thank you, I understand it perfectly now. I will do it from now on.